extras/render-passes/render-pass-camera.js

```diff
@@ -0,0 +1,181 @@
+import {
+    LAYERID_SKYBOX,
+    LAYERID_IMMEDIATE,
+    PIXELFORMAT_RGBA8,
+    ADDRESS_CLAMP_TO_EDGE,
+    FILTER_LINEAR,
+    RenderPass,
+    RenderPassColorGrab,
+    RenderPassRenderActions,
+    RenderTarget,
+    Texture
+} from "playcanvas";
+import { RenderPassBloom } from "./render-pass-bloom.js";
+import { RenderPassCompose } from "./render-pass-compose.js";
+
+class RenderPassCamera extends RenderPass {
```

Comment:
This name makes it sound like a type of camera!

If this this class implements a render pass with default post processing, shouldn't the name describe that instead?

Reply:
maybe, suggest a name?
I called it RenderPassCamera, as it represents a default render pass setup for a camera

Reply:
The correct name (if there is such a thing) would probably be `DefaultPostProcessRenderPass`. Quite a mouth (eye) full.

Reply:
but it does not just do post-processing, it renders the scene, grabs color, applies post effects .. in the future I'll add depth pre-pass, SSAO, SSR and similar. Maybe I'll call it `RenderPassFrame`